athena-neptune/src/test/java/com/amazonaws/athena/connectors/neptune/NeptuneRecordHandlerTest.java

```diff
@@ -181,7 +181,6 @@ private void buildGraphTraversal() {
                 Vertex vertex3 = tinkerGraph.addVertex(T.label, "default");
                 vertex3.property("property1", 9);
                 vertex3.property("property2", "string3");
-                vertex3.property("property3", 15.4);
                 vertex3.property("property4", true);
```

Comment:
why removing?

Reply:
We want to add test case to test the null value handling when a particular field value is null/empty

Reply:
you removed a test case here, can you add a new property to test the null/empty case?

Reply:
added new test cases